packages/terra-application-links/src/tabs/ApplicationTabs.jsx

```diff
@@ -176,19 +178,26 @@ class ApplicationTabs extends React.Component {
         collapsedTabs.push(<CollapsedTab {...tabProps} />);
       }
     });
+    const theme = this.context;
+    const tabClass = classNames(cx(
+      'tabs-container',
+      { 'is-calculating': this.isCalculating },
+      alignment,
+      theme.className,
+    ));
 
     return (
-      <div {...customProps} className={cx(['tabs-wrapper'])}>
+      <div {...customProps} className={classNames(cx('tabs-wrapper', theme.className))}>
```

Comment:
Something isn't quite right here. The theme class name is being applied to this div as well as the div below